modules/ROOT/pages/cloudhub-deployment-settings.adoc

```diff
@@ -10,6 +10,12 @@ To configure settings for deploying Anypoint Partner Manager to CloudHub, access
 | Setting | Description | Default 
 | Deployment Target Type | Target type for your Partner Manager deployment. Select *CloudHub 1.0*. | CloudHub 1.0
 | Runtime Version | Mule runtime engine version to run Partner Manager applications.| 4.6.0
+| Min Mule Version override | Minimum Mule runtime version required to deploy the runtime applications.
+
+Configuring the runtime version enables you to apply a xref:mule-runtime::feature-flagging.adoc#configure-feature-flags-for-a-mule-application[feature flagging] mechanism for Partner Manager-deployed Mule runtime applications.
+
+If you process cXML messages with <DOCTYPE> tags, you must set the `minMuleVersion` to 4.5.0 or higher. If you're migrating from an earlier version, you must perform end-to-end regression testing on your existing workflows before changing the `minMuleVersion`. | None
```

Comment:
@tasmoore - the Min Mule version config / cXML support is now part of 2.22. Please more this section.